.travis.yml

```diff
@@ -1,20 +1,15 @@
 language: python
+sudo: false
 python:
   - "2.6"
 services:
-  - elasticsearch
   - redis-server
 before_install:
-  - sudo apt-get update
-  - sudo apt-get install swig pngcrush
```

Comment:
It should be fairly simple to install pngcrush locally without sudo - it doesn't require anything fancy, you should be able to find prebuilt binaries.